### PR TITLE
Bug/so 7103 update shop image

### PR DESCRIPTION
**Pull request by ivan-yanakiev-sonect**

### CODE REVIEW

File: GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m [hunk 1]
```diff
@@ -189,15 +193,68 @@ + (void)getShopDetailsForShopId:(NSString *)placeId
...
             });
-            
             return;
         }
+        
+        if (data) {
+            UIImage *image = [UIImage imageWithData:data];
+            
+            if (image) {
+                compleionHandler(image, nil);
```
Comment (markohlebar) [inline GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m:248]: `completionHandler`

File: GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m [hunk 2]
```diff
@@ -15,6 +16,9 @@
 static NSString *addressAutocomplete = @"https://maps.googleapis.com/maps/api/place/autocomplete/json";
 static NSString *placeDetails = @"https://maps.googleapis.com/maps/api/place/details/json";
 static NSString *textSearchPlace = @"https://maps.googleapis.com/maps/api/place/textsearch/json";
+static NSString *placePhoto = @"https://maps.googleapis.com/maps/api/place/photo";
+
+static CGFloat defaultImageMaxWidth = 1000;
```
Comment (markohlebar) [inline GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m:21]: Maybe using standard values like 1024 would be better

Comment (ivan-yanakiev-sonect) [reply GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m]: Done.

File: GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m (hunk 1 above)
Comment (ivan-yanakiev-sonect) [reply GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m]: Done.

**Pull request merged**